### DISCUSSION TITLE
SX1278 @500 bd FSK Decoder

### DISCUSSION THREAD

**Discussion by lw2dtz** [jgromes/RadioLib#304] [Q&A]
Hi,

I'm trying to make a 500 Bd FSK decoder with the SX1278. I have the transmitter running
on an RFM22 in that mode and wanted to experiment using different hardware.
Is there any safe calculation to set the SX registers?

Regards.

Gustavo.

**Comment by jgromes** [comment]
Unforutantely SX127x FSK modem can only go down to 1.2 kbps, so I don't think 500 baud is possible. Apart from Si443x (same chip as RFM2x), only CC1101 can be set to 500 bps (it can go down to 25 bps). Also, SX126x comes close at the minimum bit rate of 600 bps.
